src/context/ContextListOptions.jsx

```diff
@@ -85,7 +85,7 @@ class ContextListOptions extends React.Component {
             <ul className="context-list-options" ref="contextListOptions">
                 {contexts.map((context, index) => {
                     return <ContextItem
-                        key={context.key}
+                        key={index}
```

Comment:
[DP3] We really shouldn't be using just index as a key. Won't affect this list so much since it's relatively static, but we should probably use something more unique to the context options.

Reply:
I originally used context.key as you can see so maybe we want to switch it back to that?

Reply:
I was getting errors that the keys were not unique with key={context.key}. It happened specifically I think on the #adverse event when expanding the text for #toxicity